templates/default/less/variables.less

```diff
@@ -90,16 +93,23 @@
 @border-radius-large:       0px;
 @border-radius-small:       0px;
 
+// This controls the heights of the header components.
+// controls height of the "topbar" (that gray thingy on top of the page in the standard).
+@il-top-bar-height:			40px;
+// controls the height of the "main header" (that white thingy with the logo and the main menu in it).
+@il-main-header-height:		80px;
```

Comment:
There are no agreed upon guidelines yet. Currently we use the il- prefix to make clear that this is an ILIAS and not a bootstrap variable. 

Concerning naming: We do not have agreed upon naming. In: [https://docs.google.com/document/d/1JoInU0-tkRHDOMjMTNziV-hJb6ypDCI1zT9OGtAThZk/edit#heading=h.sndryvcwtyj6] we propose a header section with an application branding, top navigation and main menu. This will be surely discussed an might be changed. So I think the proposed of @klees naming works for the moment.